### ISSUE TITLE
Google sheet crash on update

### ISSUE THREAD

**Issue by dokime7** [n8n-io/n8n#984]
**Describe the bug**
Since update to 0.84.1, Google sheet update return an error.

**To Reproduce**
1. Add Google sheet node
2. Select operation : Update
3. Fill credentials and others fields
4. Run it

**Environment (please complete the following information):**
 - OS: Ubuntu 16.04.6 LTS
 - n8n Version 0.84.1
 - Node.js Version that's come with docker image 0.84.1

**Additional context**
Error returned by Google sheet node:
```
ERROR: Cannot read property 'match' of undefined
TypeError: Cannot read property 'match' of undefined
    at GoogleSheet.updateSheetData (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Sheet/GoogleSheet.js:112:40)
    at Object.execute (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/Google/Sheet/GoogleSheets.node.js:663:42)
    at Workflow.runNode (/usr/local/lib/node_modules/n8n/node_modules/n8n-workflow/dist/src/Workflow.js:489:37)
    at /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/src/WorkflowExecute.js:369:62
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```
Operation Read is working as expected.

**Comment by dokime7** [comment]
I revert to 0.83.0 version and it's working here.

**Comment by dokime7** [comment]
Tested with the last one 0.84.3, the problem persist on it.

**Comment by RicardoE105** [comment]
@dokime7  Thanks for letting us know.  I am investigating this, it seems this commit breaks the node https://github.com/n8n-io/n8n/commit/9e97722c6365a5dc8a607582833b06a9b78ba44d

**Comment by RicardoE105** [comment]
@dokime7 just sent a PR fixing the issue, we will let you know when is released. Again, thanks for letting us know.

**Comment by dokime7** [comment]
Great, thanks for reactivity :)

**Comment by janober** [comment]
Thanks a lot @RicardoE105!

Got released with n8n@0.84.4

Can you please retry with that version.

**Comment by dokime7** [comment]
Yes! It's working with 0.84.4.
Thanks a lot!
